src/tests/suite/daos_rebuild_simple.c

```diff
@@ -80,9 +85,10 @@ reintegrate_with_inflight_io(test_arg_t *arg, daos_obj_id_t *oid,
 					 0, arg->myrank);
 	inflight_oid = dts_oid_set_rank(inflight_oid, rank);
 
+#if 0
```

Comment:
comment?

Reply:
the comment is above?

Reply:
Ok, but is it easy to correlate if 0 ?  Maybe if it was #ifdef DAOS_8775 or something so they were correlated

Reply:
oh, sure. Anyway, I have to remove this soon, since we need support inflight I/O during reintegration for 2.2 soon anyway.